doc/related_projects.rst

```diff
@@ -50,6 +50,12 @@ enhance the functionality of scikit-learn's estimators.
   includes a replacement for ``GridSearchCV`` or ``RandomizedSearchCV`` to do
   cross-validated parameter search using any of these strategies.
 
+- `Neuraxle <https://github.com/Neuraxio/Neuraxle>`_
+  A Python Framework for Hyperparameter Tuning and Clean Machine Learning. Let
+  your pipeline steps have hyperparameter spaces. Enable data checkpoints to
+  cut duplicate calculations. Serialize your models for production. Go from
+  research to production environment and deploy easily.
```

Comment:
We need to change it to read less as a slogan and more as neutral description of the project :)

I think the description at from the readme might be a good start,

> is a Machine Learning (ML) library for building neat pipelines, providing the right abstractions to both ease research, development, and deployment of your ML applications.

with maybe a sentence or two about data checkpoints and model serialization?